cypress/support/config/services.js

```diff
@@ -873,6 +873,30 @@ module.exports = {
       mediaAssetPage: { path: undefined, smoke: false },
     },
   },
+  scotland: {
+    font: undefined,
+    isWorldService: false,
+    variant: 'default',
+    pageTypes: {
+      articles: {
+        path:
+          Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
+            ? undefined
+            : '/scotland/articles/c1111111111o',
+        smoke: false,
+      },
+      errorPage404: {
+        path:
+          Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
+            ? undefined
+            : '/scotland/articles/cabcdefghijo',
+        smoke: false,
+      },
+      frontPage: { path: undefined, smoke: false },
```

Comment:
Can we add `/scotland` to be available locally in Simorgh?

Reply:
There's no expectation we will ever serve the /scotland homepage - that's part of the BBC Homepage product.

Reply:
Okay, thanks.